chapter_introduction/index_vn.md

```diff
@@ -2247,7 +2259,10 @@ were essentially "rediscovered" in the past decade,
 after laying comparatively dormant for considerable time.
 -->
 
-*dịch đoạn phía trên*
+Sự thật là RAM đã không phát triển cùng với sự lớn lên của dữ liệu.
+Đồng thời, sự tiến bộ trong năng lực tính toán đã vượt lên sự sẵn có của dữ liệu.
+Điều này nghĩa là các mô hình thống kê cần phải trở nên hiệu quả hơn về bộ nhớ (điều này thường đặt được bằng cách thêm các thành phần phi tuyến) đồng thời có thể tập trung thời gian cho việc tối ưu các tham số bởi chi phí tính toán tăng. Kéo theo đó, <sweek spot> trong học máy và thống kê dịch chuyển từ các mô hình tuyến tính (tổng quát) và các phương pháp hạt nhân (_kernel methods_) sang các mạng nơ-ron sâu. <!-- thêm nơ-ron để tránh nhầm lẫn -->
```

Comment:
- kernel method == `phương pháp hạt` được không nhỉ? 'hạt nhân' nghe cứ liên tưởng nuclear.
- sweet spot không hoàn toàn tương đương `tiêu điểm` nhưng em thấy trong ngữ cảnh này vẫn hợp.
```suggestion
Điều này nghĩa là các mô hình thống kê cần phải trở nên hiệu quả hơn về bộ nhớ (điều này thường đạt được bằng cách thêm các thành phần phi tuyến) đồng thời có thể tập trung thời gian cho việc tối ưu các tham số bởi khả năng tính toán đã tăng. Kéo theo đó, tiêu điểm trong học máy và thống kê đã dịch chuyển từ các mô hình tuyến tính (tổng quát) và các phương pháp hạt nhân (_kernel methods_) sang các mạng nơ-ron sâu. <!-- thêm nơ-ron để tránh nhầm lẫn -->
```